tests/social-share-button.test.js

```
@@ -0,0 +1,568 @@
+
+    instance.openModal();
+
+    // Click the container itself, not a button
+    const container = instance.modal.querySelector(".social-share-platforms");
+    container.click();
+
+    expect(shareSpy).not.toHaveBeenCalled();
+  });
+
+  it("should use fewer event listeners than the number of platforms", () => {
+    const platformCount = 8;
```

ESLint

[error] 'platformCount' is assigned a value but never used.

(no-unused-vars)